graphql-dynamodb-manager-core/src/main/java/com/fleetpin/graphql/dynamodb/manager/DynamoDb.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.fleetpin.graphql.dynamodb.manager;
+
+import com.fleetpin.graphql.dynamodb.manager.table.Table;
+import org.dataloader.DataLoader;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+public interface DynamoDb {
```

Comment:
should rename this to something else not sure what since think we already used database

Reply:
I don't have the slightest clue on what to rename it to.
It should be alright because the InMemoryDynamoDb uses the DynamoItem as the value in the unlying ConcurrentHashMap.